src/java.base/share/classes/java/lang/Double.java

```diff
@@ -680,12 +680,12 @@ public static String toHexString(double d) {
      *       "[fFdD]?))" +
      *       "[\\x00-\\x20]*");// Optional trailing "whitespace"
      *
-     *  if (Pattern.matches(fpRegex, myString))
+     *  if (Pattern.matches(fpRegex, myString)) // @link substring="Pattern.matches" target ="java.util.regex.Pattern#matches"
```

Comment:
It can be done with a snippet region like this:

```
     *  // @link region substring="Pattern.matches" target ="java.util.regex.Pattern#matches"
     *  if (Pattern.matches(fpRegex, myString))
     *      Double.valueOf(myString); // Will not throw NumberFormatException
     * // @end
```

Reply:
Yes, that works. The alternative is to terminate the comment line with a colon (:) and that applies markup tag to the line that follows. In this case, it would be:

 ```
// @link substring="Pattern.matches" target ="java.util.regex.Pattern#matches" :
```